app/main.py

```diff
@@ -2,4 +2,63 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self, name: str, weight: int, coords: list = None
```

Comment:
specify list annotation 
```suggestion
            self, name: str, weight: int, coords: list[int] = None
```